services/introspection/local.go

```diff
@@ -231,7 +224,7 @@ func pluginsToPB(plugins []*plugin.Plugin) []*api.Plugin {
 			Type:         p.Registration.Type.String(),
 			ID:           p.Registration.ID,
 			Requires:     requires,
-			Platforms:    platforms,
+			Platforms:    plats,
```

Comment:
nit: maybe merge line 192 here directly (`Platforms: platforms.ToProto(p.Meta.Platforms)`)

Reply:
That's a better idea, changing!